tests/hardhat/Shortfall.ts

```diff
@@ -41,12 +43,12 @@ let fakePriceOracle: FakeContract<PriceOracle>;
 
 let riskFundBalance = "10000";
 const minimumPoolBadDebt = "10000";
-let poolId;
+let poolAddress;
 
 async function mineNBlocks(n: number) {
   for (let index = 0; index < n; index++) {
     await ethers.provider.send("evm_mine", [Date.now() * 1000]);
-    await sleep(100);
+    await sleep(10);
```

Comment:
Why do we need `sleep` here?

Reply:
Maybe we could use hardhat network method for this?

```
import { mine } from "@nomicfoundation/hardhat-network-helpers";

// instantly mine 1000 blocks
await mine(1000);
```

Reply:
ok let me try this